<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QDialog" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo-Project</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QGroupBox" name="regBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>225</y>
     <width>281</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
            QGroupBox {
              border: 1px solid #E0E0E0;
              border-radius: 8px;
              margin-top: 0px;
              padding-top: 25px;
              font: bold 12pt &quot;Segoe UI&quot;;
              color: black;
              background: white;
            }
            QGroupBox::title {
			 subcontrol-origin: margin;
              left: 5px;
			 top: 5px;
            }</string>
   </property>
   <property name="title">
    <string>Регистрация</string>
   </property>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>5</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">            QPushButton {
              background: #E0E0E0;
              color: #616161;
              border-radius: 6px;
              font: bold 12pt &quot;Segoe UI&quot;;
            }
            QPushButton:hover {
              background: #BDBDBD;
            }</string>
    </property>
    <property name="text">
     <string>⬅</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>256</width>
      <height>136</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">color: black;</string>
           </property>
           <property name="text">
            <string>Введите логин</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">color: black;</string>
           </property>
           <property name="text">
            <string>Введите пароль</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">border-radius: 3px;
color: black;</string>
           </property>
           <property name="text">
            <string>Повторите пароль</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="loginLine">
           <property name="styleSheet">
            <string notr="true">color: black;</string>
           </property>
           <property name="maxLength">
            <number>12</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passLine">
           <property name="styleSheet">
            <string notr="true">color: black;</string>
           </property>
           <property name="maxLength">
            <number>12</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="secpassLine">
           <property name="styleSheet">
            <string notr="true">background-color: white;
color: black;</string>
           </property>
           <property name="maxLength">
            <number>12</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="regButton">
       <property name="styleSheet">
        <string notr="true">padding: 8px 10px;
font-size: 10pt;
color: black;</string>
       </property>
       <property name="text">
        <string>Зарегистрироваться</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
